site/package.json

```
@@ -37,6 +37,7 @@
     "@lingui/cli": "^3.13.0",
     "@lingui/loader": "^3.13.0",
     "@lingui/macro": "^3.13.0",
+    "@next/bundle-analyzer": "^12.3.1",
```

Can you also upgrade the one in `app` to match 12.3.1, looks like its 12.0.1 right now
```suggestion
    "@next/bundle-analyzer": "^12.3.1",
```